ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user events inside the side bar.
+@objc(MSFSideBarDelegate)
+public protocol SideBarDelegate {
+    /// Called after the view representing `TabBarItem` is selected.
+    @objc optional func sideBar(_ sideBar: SideBar, didSelect item: TabBarItem, fromTop: Bool)
+
+    /// Called after the avatar view is tapped in the side bar.
+    @objc optional func sideBar(_ sideBar: SideBar, didActivate avatarView: AvatarView)
+}
+
+/// View for a vertical side bar that can be used for app navigation.
+/// Optimized for horizontal regular + vertical regular size class configuration. Prefer using TabBarView for other size class configurations.
+@objc(MSFSideBar)
+open class SideBar: UIView {
```

Comment:
>SideBar [](start = 11, length = 7)

if this uses tabBarItems mostly, why not call it SideTabBar? Just wonder if it'll be confused with the new iOS14 SideBar